packages/contracts/contracts/L1/MockL2OutputOracle.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity >=0.8.10;
  * @title MockL2OutputOracle
  */
 contract MockL2OutputOracle {
```

Comment:
I see this is a `MockL2OutputOracle` - it doesn't have the concept of auth for who can submit batches, is this functionality being tracked/specified?

Reply:
This won't stay a mock. I hope it's tracked but haven't checked!

As for speccing the auth, I think there won't be much more to say at first than "there is an address owning this". Anything more you'd like to suggest there?

Reply:
Whoever submits batches should have a deposit that is slashable if the fraud proof executes and the proposer is shown to have submitted fraudulent data. Right now it looks like anybody can append an L2 output with no mechanism to prevent liars

Reply:
That will be added when Cannon goes live, right now it's permissioned (which is what I meant by "there is an address owning this", should have expressed that better).